runtime/compiler/env/VMJ9.cpp

```diff
@@ -4722,6 +4722,14 @@ TR_J9VMBase::getOverflowSafeAllocSize()
    return result;
    }
 
+void
+TR_J9VMBase::unsupportedByteCode(TR::Compilation * comp, U_8 opcode)
+   {
+   char errMsg[40];
+   sprintf(errMsg, "bytecode %d not supported by JIT", opcode);
```

Comment:
snprintf with some error checking would be much safer here

Reply:
should we enhance the unknonByteCode case like this as well - it seems a much better error message since you are "in the area"

Reply:
Updated the code according to comments. Thanks!

Reply:
Please ingore the comment about breakpoint in `unknownByteCode` for now. I will change that comment when adding assert_fatal in a separate PR.